.github/workflows/macOS-CMake-toolchain.yml

```diff
@@ -0,0 +1,88 @@
+name: macOS (CMake toolchain)
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    env:
+      # Hardcoded date selecting the latest toolchain from https://swift.org/download/#snapshots.
+      toolchain-date: 2021-01-04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set environment variables
+        run: |
```

Comment:
This seems like a large set of variables to define.  I assume that most of these are only needed for packaging?  It might make sense to make the packaging-only ones be set in the environment of the packaging.

Reply:
Thanks for the pointing this out! I moved packaging-only variables to the specific packaging step, the list of variables at the top is more reasonable now.